src/cr/cube/measures/scale_means.py

```diff
@@ -19,15 +19,17 @@ def data(self):
         contents = self._inner_prod(table, self.values)
 
         if self._cube.has_mr and not self._cube.is_double_mr:
-            axis = 1 - self._cube.mr_dim_ind
+            # axis = 1 - self._cube.mr_dim_ind
+            # axis = self._cube.dim_types[-2:].index('categorical')
+            axis = self._cube.dim_types.index('categorical')
             return np.sum(contents, axis) / np.sum(table, axis)
```

Comment:
I commented to this end on the exporter PR for this change, I think we should be able to specify which axis the means are to be calculated for in crunch-cube and have exporter explicitly ask for the axis it wants/needs rather than having the special cases here. 

In the circumstance where an axis is requested that is ineligible for means (a multiple response, or a categorical without numeric values), you can either return all `NaNs` or an empty array.